src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2844,6 +2845,8 @@ private void GenerateHelperFunctions(TextWriter writer)
 
     End {{ {7} }}
 
+    Clean {{ {8} }}
+
```

Comment:
I took a close look at the changes in `ImplicitRemotingCommands.cs`, and it turns out we don't need to add the `Clean` block in the generated script at all. This is because `$steppablePipeline` in the script only contains the `Invoke-Command` cmdlet. `Invoke-Command` is a binary cmdlet which doesn't support the `clean` block (`clean` block is always a non-op for binary cmdlet).

So, I will revert the changes in this file.

Reply:
Changed in this file reverted, except for an indentation change (it's just look so much better after changing it :)).